docs/components/index.md

```diff
@@ -116,38 +116,79 @@ Data binding works with [DateTime](https://docs.microsoft.com/dotnet/api/system.
 }
 ```
 
+The `bind` Tag Helper uses the `format-value` attribute to apply the date format to the `value` node of the `input` element. The format is also applied by the value handler of the `onchange` node.
+
 **Component attributes**
 
 Binding also recognizes component attributes, where `bind-{property}` can bind a property value across components.
 
-The following parent component uses `ChildComponent` and binds the value `1979` from `ParentYear` to the child component's `Year` property:
+The following component uses `ChildComponent` and binds the value `1978` from the `ParentYear` property to the child component's `Year` property:
```

Comment:
binds the `ParentYear` parameter from the parent to the `Year` parameter on the child component.